addons/web/static/src/views/calendar/filter_panel/calendar_filter_panel.js

```diff
@@ -19,6 +20,14 @@ export class CalendarFilterPanel extends Component {
         });
         this.addDialog = useOwnedDialogs();
         this.orm = useService("orm");
+
+        useSetupView({
+            getLocalState: () => {
+                const sections = this.props.model.filterSections;
+                const allFilter = sections.length > 0 ? sections[0].filters?.find((f) => f.type === "all") : false;
+                return allFilter ? { allFilterState: allFilter.active } : {};
+            },
+        });
```

Comment:
We are not in a view, don't use `useSetupView` here. Set the state in `CalendarModel.updateFilter` instead.